server/src/test/java/io/crate/integrationtests/ColumnPolicyIntegrationTest.java

```diff
@@ -163,11 +166,13 @@ public void testInsertArrayIntoDynamicTable() throws Exception {
     }
 
     @Test
+    // Ensure that PKLookup is hit.
+    @UseRandomizedOptimizerRules(alwaysKeep = {MergeFilterAndCollect.class, OptimizeCollectWhereClauseAccess.class})
     public void testInsertDynamicObjectArray() throws Exception {
```

Comment:
Without this PKlookup sometimes passes even on master even though it has a bug.

Thanks @jeeminso for pointing it out!